src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  die() {
+    Animal.alive = Animal.alive.filter((animal) => animal !== this);
+  }
```

Comment:
The `die` method in the Animal class filters the list of live animals by the health of each instance. 
This works, but depending on this method in the Herbivore class can complicate the code. 

Better to have this method at the Animal class level to avoid such problems.
Example: 
```suggestion
  static removeDeadAnimals() {
    Animal.alive = Animal.alive.filter(animal => animal.health > 0);
  }
```